Graphics/Master Graphics Installer.event

```diff
@@ -24,9 +24,13 @@ ORG 0x5926F4+(0x80*HuntersBowIconID)
 #incext Png2Dmp "HuntersBow.png"
 ORG 0x5926F4+(0x80*RunicBowIconID)
 #incext Png2Dmp "RunicBow.png"
+ORG 0x5926F4+(0x80*PiercingBowIconID)
+#incext Png2Dmp "PiercingBow.png"
 
+ORG 0x5926F4+(0x80*0x79)
```

Comment:
Please use a definition for this.

Reply:
the other replacing-vanilla-icon example (blitz axe) used a constant. copied consistency.